sections/header.liquid

```diff
@@ -680,6 +697,10 @@
     }
 
     reset() {
+      if (this.getAttribute('data-sticky-type') === 'always') {
+        this.header.classList.remove('animate');
+        return;
+      }
```

Comment:
Actually, the purpose of this line is to avoid removing the 'shopify-section-header-sticky' below.

Because it's always sticky, we shouldn't ever be removing this class so my idea was to do an early return here. 
But I just realized that we shouldn't be removing this `animate` class either? If it's always sticky :P

Probably a 
```js
      if (this.getAttribute('data-sticky-type') === 'always') return;
```
will be better.